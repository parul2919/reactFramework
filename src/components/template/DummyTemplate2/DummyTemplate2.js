import React from "react";
import styled from "styled-components";
import styles from "./DummyTemplate2.style";
import MemoizedFooter from "../../organism/Footer/Footer";
import MemoizedHeader from "../../organism/Header/Header";

const DummyTemplate2 = (props) => {
  const { className } = props;
  return (
    <>
      <MemoizedHeader />
      <div className={`${className} dummy-template`}>
        Here will have main content
      </div>
      <MemoizedFooter />
    </>
  );
};

export default styled(DummyTemplate2)`
  ${styles}
`;
export { DummyTemplate2 };
