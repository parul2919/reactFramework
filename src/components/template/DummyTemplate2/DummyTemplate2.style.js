import { css } from "styled-components";

export default css`
  &.dummy-template {
    height: 100vh;
  }
`;
