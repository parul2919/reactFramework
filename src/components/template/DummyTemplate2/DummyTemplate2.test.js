import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter as Router } from "react-router-dom";
import DummyTemplate2 from "./DummyTemplate2";

describe("<Component /> spec", () => {
  it("renders the component", () => {
    const view = render(
      <Router>
        <DummyTemplate2 />
      </Router>
    );
    expect(view).toMatchSnapshot();
  });
});
