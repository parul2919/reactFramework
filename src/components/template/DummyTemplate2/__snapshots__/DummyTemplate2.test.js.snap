// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Component /> spec renders the component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header sc-gsnTZi bgEuUO"
      >
        <nav
          class="navbar navbar-expand"
        >
          <a
            class="navbar-brand "
            href="/"
          >
            <img
              alt="logo-img"
              src="test-file-stub"
              width="150"
            />
          </a>
          <div
            class="collapse navbar-collapse"
          >
            <ul
              class="navbar-nav ml-auto "
            >
              <a
                class="nav-link"
                href="/"
                style="text-decoration: underline;"
                target="_self"
              >
                Demo Link 1
              </a>
              <a
                class="nav-link"
                href="/demoLink2"
                style="text-decoration: none;"
                target="_self"
              >
                Demo Link 2
              </a>
              <a
                class="nav-link"
                href="/demoLink3"
                style="text-decoration: none;"
                target="_self"
              >
                Demo Link 3
              </a>
            </ul>
          </div>
        </nav>
      </header>
      <div
        class="sc-dkzDqf fjderG dummy-template"
      >
        Here will have main content
      </div>
      <section
        class="footer sc-bczRLJ eRSPWO"
      >
        <nav
          style="max-width: 1440px; margin: auto;"
        >
          <a
            class="footer-logo"
            href="/"
          >
            <img
              alt="logo-img"
              src="test-file-stub"
              width="150"
            />
          </a>
        </nav>
        <div
          class="disclaimer-text text-center"
        >
          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ips um has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <header
      class="header sc-gsnTZi bgEuUO"
    >
      <nav
        class="navbar navbar-expand"
      >
        <a
          class="navbar-brand "
          href="/"
        >
          <img
            alt="logo-img"
            src="test-file-stub"
            width="150"
          />
        </a>
        <div
          class="collapse navbar-collapse"
        >
          <ul
            class="navbar-nav ml-auto "
          >
            <a
              class="nav-link"
              href="/"
              style="text-decoration: underline;"
              target="_self"
            >
              Demo Link 1
            </a>
            <a
              class="nav-link"
              href="/demoLink2"
              style="text-decoration: none;"
              target="_self"
            >
              Demo Link 2
            </a>
            <a
              class="nav-link"
              href="/demoLink3"
              style="text-decoration: none;"
              target="_self"
            >
              Demo Link 3
            </a>
          </ul>
        </div>
      </nav>
    </header>
    <div
      class="sc-dkzDqf fjderG dummy-template"
    >
      Here will have main content
    </div>
    <section
      class="footer sc-bczRLJ eRSPWO"
    >
      <nav
        style="max-width: 1440px; margin: auto;"
      >
        <a
          class="footer-logo"
          href="/"
        >
          <img
            alt="logo-img"
            src="test-file-stub"
            width="150"
          />
        </a>
      </nav>
      <div
        class="disclaimer-text text-center"
      >
        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ips um has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
